### PR TITLE
Clarify the status of foliage -- not a separate chain

### PR DESCRIPTION
**Pull request by danieljperry**

### CODE REVIEW

File: docs/consensus/foliage.md [hunk 1]
```diff
@@ -4,22 +4,22 @@ slug: /consensus-foliage
...
 
-To prevent a foliage re-org, honest farmers only create one foliage block per block. As soon as one honest farmer has added a foliage block, the foliage becomes impossible to re-org beyond that height with the same PoSpace, since that farmer will not sign again with the same PoSpace.
+To prevent a foliage re-org, honest farmers only create one set of foliage per block. As soon as one honest farmer has added foliage to a block, the foliage becomes impossible to re-org beyond that height with the same PoSpace, since that farmer will not sign again with the same PoSpace.
 
-Furthermore, blocks like B3, which come parallel with another foliage block (B2), do not have to sign the previous foliage block, since they do not necessarily have enough time to see it.
+Furthermore, blocks like B3, which come parallel with another the foliage of another block (B2), do not have to sign the previous block's foliage, since they do not necessarily have enough time to see it.
```
Comment (hoffmang9) [inline docs/consensus/foliage.md:22]: Wording is off here

Comment (BrandtH22) [reply docs/consensus/foliage.md:22]: Looks like the word "another" needs to be removed

**Comment by BrandtH22** [comment]
@danieljperry  As a heads up, I have also started on some reworks to this documentation and have connected with almogd for more clarification. 
I will likely be in touch via keybase once I have more info :+1:

**Comment by BrandtH22** [comment]
@danieljperry this is looking great!! I passed along the PR to Almog to review also, if there are no issues I will approve and we will be good to merge. Thank you!

**Review by hoffmang9 — APPROVED** [review]
lgtm

**Comment by BrandtH22** [comment]
No issues have been identified

**Pull request merged**